### ISSUE TITLE
Random file name

### ISSUE THREAD

**Issue by nnever** [JoeGandy/ShareX-Custom-Upload#31]
It would be cool if there was a config option to generate a random name on the php end instead of providing the name from your sharex client. It looks like there's no checking if the file already exists on the server so you can't use sharex to randomize characters because you'll eventually have conflicts. A lot of other uploaders already have this, but this one seems to be the best besides that issue.

**Comment by JoeGandy** [comment]
I agree it should be in there.  My initial thoughts were sharexs timestamps
were good enough, adding in milliseconds makes it basically impossible to
overlap.

But yeah this feature isn't in more of of lazyness and 'it works well
enough to attitude' :)

**Comment by Zarken-Dev** [comment]
Is this going to be added in or?

**Comment by Zarken-Dev** [comment]
> @Zarken-Dev, feel free to make a pull request! :D

Thanks for the offer :) i managed to code it myself :D

**Comment by Natfan** [comment]
@Zarken-Dev If you could share the code with us, we can look into adding it to the master branch? :)

**Comment by Zarken-Dev** [comment]
> @Zarken-Dev If you could share the code with us, we can look into adding it to the master branch? :)

Enjoy: https://pastebin.com/ivgUw3zY - Thats the upload.php and I just used code found from other places and made it work.

**Comment by DJkikisa** [comment]
Sorry for too late answer, i had no spare time.
